### DISCUSSION TITLE
los códigos fuente ya no se permiten editar

### DISCUSSION THREAD

**Discussion by aguerra72** [codefori/vscode-ibmi#1670] [IBM i]
Con la versión VSCODE 1.84.2 
los códigos fuente ya no se permiten editar
cómo se observa en la imagen  [https://prnt.sc/WN4gkh3VKpEK](https://prnt.sc/WN4gkh3VKpEK) , quedan con un candado y no permiten cambios

en el IBM i, la biblioteca los archivos fuente y los miembros tienen permisos de PUBLIC * ALL

los settings del filtro se encuentran sin la marca de protegido como se ve en la imagen  [https://prnt.sc/5WyoPufUCxmp](https://prnt.sc/5WyoPufUCxmp)

los settings del complemento codefori tambien estan sin la marca READONLY como se ven en la imagen
[https://prnt.sc/3Wm3pAVSk_ch](https://prnt.sc/3Wm3pAVSk_ch)

**Comment by worksofliam** [comment]
Comparta el contenido del canal de salida Code for IBM i despues de que la lista no aparezca por favor

<img width="939" alt="image" src="https://user-images.githubusercontent.com/3708366/178739532-03c8ad30-84de-4698-9904-65938f2ef651.png">

**Reply by aguerra72** [reply]
no aparece nada, adjunto imagen
[https://prnt.sc/TD5xvwCPovge](https://prnt.sc/TD5xvwCPovge)
